client/stylesheets/_modal.scss

```diff
@@ -233,14 +233,55 @@
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	History Details Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
 
+/*~~ Button End ~~*/
+
+#profileHistoryFeedback {
+	width: 100% !important;
+    background-color: #f8504b !important;
+    padding: 15px 0 !important;
+    text-align: center !important;
+    border-radius: 4px !important;
+    color: #fff !important;
+	text-decoration: none !important;
+}
 
+#profileHistoryFeedback:hover {
+	background-color: #f73c37 !important;
+}
 
+#profileHistoryFeedback a {
+	color: #fff !important;
+	text-decoration: none !important;
+	padding: 15px 140px !important;
+}
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	END END END END END END
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Page Name Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~ Button ~~~~*/
 
+.modEditListingTrigger {
+	background-color: #fff !important;
+    width: 100% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, padding, text-align, width